net/crowdsec/Makefile

```diff
@@ -0,0 +1,164 @@
+
+CWD_SYSTEM:=openwrt
+CWD_BUILD_VERSION?=$(PKG_SOURCE_VERSION)
+CWD_BUILD_GOVERSION:=$(shell go version | cut -d " " -f3 | sed -E 's/[go]+//g')
+CWD_BUILD_CODENAME:=alphaga
+CWD_BUILD_TIMESTAMP:=$(shell date +%F"_"%T)
+CWD_BUILD_TAG:=openwrt
+
+CWD_VERSION_PKG:=github.com/crowdsecurity/crowdsec/pkg/cwversion
+
+GO_PKG:=github.com/crowdsecurity/crowdsec
+
+GO_PKG_LDFLAGS_X:=$(CWD_VERSION_PKG).Version=$(CWD_BUILD_VERSION) \
+	$(CWD_VERSION_PKG).System=$(CWD_SYSTEM) \
+	$(CWD_VERSION_PKG).BuildDate=$(CWD_BUILD_TIMESTAMP) \
+	$(CWD_VERSION_PKG).Codename=$(CWD_BUILD_CODENAME)  \
+	$(CWD_VERSION_PKG).Tag=$(CWD_BUILD_TAG) \
+	$(CWD_VERSION_PKG).GoVersion=$(CWD_BUILD_GOVERSION)
+
+GO_PKG_INSTALL_ALL:=1
+
+ifeq ($(ARCH),$(filter $(ARCH), mips mipsel mips64 mips64el))
+	TARGET_LDFLAGS+= -static
+endif
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/crowdsec/Default
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=CrowdSec detection engine
+  URL:=https://crowdsec.net/
+endef
+
+define Package/crowdsec
+$(call Package/crowdsec/Default)
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/golang-crowdsec-dev
+$(call Package/crowdsec/Default)
+$(call GoPackage/GoSubMenu)
+  TITLE+= (source files)
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+  PKGARCH:=all
+endef
+
+define Package/crowdsec/Default/description
+  Crowdsec - An open-source, lightweight agent to detect
+  and respond to bad behaviours.
+  It also automatically benefits from our global
+  community-wide IP reputation database
+endef
+
+define Package/crowdsec/description
+$(call Package/crowdsec/Default/description)
+
+  This package contains the main program.
+endef
+
+define Package/golang-crowdsec-dev/description
+$(call Package/crowdsec/Default/description)
+
+  This package provides the source files for the program.
+endef
+
+define Package/crowdsec/install
+	$(call GoPackage/Package/Install/Bin,$(1))
+
+	$(INSTALL_DIR) $(1)/etc/crowdsec
+	$(INSTALL_DIR) $(1)/etc/crowdsec/scenarios
+	$(INSTALL_DIR) $(1)/etc/crowdsec/postoverflows
+	$(INSTALL_DIR) $(1)/etc/crowdsec/collections
+	$(INSTALL_DIR) $(1)/etc/crowdsec/patterns
+	$(INSTALL_DIR) $(1)/etc/crowdsec/hub
+
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/config.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/dev.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/user.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/acquis.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/profiles.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/simulation.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/local_api_credentials.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/online_api_credentials.yaml \
```

Comment:
looks like CrowdSec is still managing to enhance the system/user config files !
So for now, It will be better to manage and add all files from /etc/crowdsec to conffiles...

will add this in next commit.

Reply:
modified, thanks